.ci.yaml

```diff
@@ -3215,7 +3215,24 @@ targets:
       - bin/**
       - .ci.yaml
 
-  - name: Mac verify_binaries_codesigned
+  - name: Mac_x64 verify_binaries_codesigned
+    bringup: true
+    enabled_branches:
+      - flutter-\d+\.\d+-candidate\.\d+
+    recipe: flutter/flutter
+    timeout: 60
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "xcode", "version": "14a5294e"}
```

Comment:
this cipd package already exists for arm?

Reply:
Yes, it's fat and will work on either.

Reply:
I shudder to think how big a fat xcode package is.